pkg/virt-launcher/virtwrap/device/hostdevice/hostdev.go

```diff
@@ -95,6 +99,27 @@ func createPCIHostDevice(hostDeviceData HostDeviceMetaData, hostPCIAddress strin
 	return domainHostDevice, nil
 }
 
+func createMDEVHostDeviceWithDisplay(hostDeviceData HostDeviceMetaData, mdevUUID string) (*api.HostDevice, error) {
+	mdev, err := createMDEVHostDevice(hostDeviceData, mdevUUID)
+	if err != nil {
+		return mdev, err
+	}
+	if hostDeviceData.VirtualGPUOptions != nil {
+		if hostDeviceData.VirtualGPUOptions.Display != nil {
+			if *hostDeviceData.VirtualGPUOptions.Display.Enabled {
```

Comment:
Can't `hostDeviceData.VirtualGPUOptions.Display.Enabled` be nil here?

Reply:
It can, but leads to exactly the same result as setting false, and this scenario is already covered, so I'm not sure why would we need another test?
Keep in mind that this API is only needed to be used in cases where the users want to explicitly turn off the display or the ramFB option for that display.

Reply:
If it's nil won't that line segfault? In which case the fact that all tests are passing probably indicated that we don't test that possibility.

Reply:
ah, true.

Reply:
@jean-edouard  Updated, thanks.